tools/compile_tf_graph.py

```diff
@@ -289,7 +402,7 @@ def final_op(self):
         x.placeholder = value
         x = x.copy_compatible_to(self.var_data_shape)
         value = x.placeholder
-      return tf.assign(self.var, value, name="final_state_var_%s" % self.name).op
+      return tf.assign(self.var, value, name="final_state_var_%s" % self.name, validate_shape=False).op
```

Comment:
Why is that `validate_shape` needed? It should not be needed I think.

Reply:
same